apps/chat-e2e/src/tests/sharedFilesAttachments.test.ts

```diff
@@ -0,0 +1,477 @@
+      /*'EPMRTC-4135,'*/
+      'EPMRTC-4155',
+      'EPMRTC-4123',
+      'EPMRTC-3116',
+      'EPMRTC-3122',
+      'EPMRTC-4164',
+    );
+    let imageUrl: string;
+    let imageUrl2: string;
+    let imageInConversationInFolderUrl: string;
+    let specialCharsImageUrl: string;
+    //TODO EPMRTC-4135 blocked by the #1076
+    // let imageInFolderUrl2: string;
+    let shareByLinkResponse: ShareByLinkResponseModel;
+    let shareFolderByLinkResponse: ShareByLinkResponseModel;
+    let defaultModel: ModelIds;
+    let conversationInFolder: Conversation;
+    //TODO EPMRTC-4135 blocked by the #1076
+    // let conversationToMove: Conversation;
+    const folderName = 'Folder with conversation';
+    const specialCharsFolder = `Folder ${ExpectedConstants.allowedSpecialChars}`;
+    let conversationWithSpecialChars: Conversation;
+
+    let conversationWithTwoResponses: Conversation;
+
+    await localStorageManager.setChatCollapsedSection(
+      CollapsedSections.Organization,
+    );
+
+    await dialTest.step(
+      'Upload image file to a conversation and prepare conversation with attachments in response',
+      async () => {
+        defaultModel = ModelIds.GPT_4_O;
+        await fileApiHelper.deleteAllFiles();
+        imageUrl = await fileApiHelper.putFile(
+          Attachment.sunImageName,
+          API.modelFilePath(defaultModel),
+        );
+        imageUrl2 = await fileApiHelper.putFile(
+          Attachment.cloudImageName,
+          API.modelFilePath(defaultModel),
+        );
+        imageInConversationInFolderUrl = await fileApiHelper.putFile(
+          Attachment.flowerImageName,
+          API.modelFilePath(defaultModel),
+        );
+        specialCharsImageUrl = await fileApiHelper.putFile(
+          Attachment.specialSymbolsName,
+          specialCharsFolder,
+        );
+
+        //TODO EPMRTC-4135 blocked by the #1076
+        // imageInFolderUrl2 = await fileApiHelper.putFile(
+        //   Attachment.heartImageName,
+        //   API.modelFilePath(defaultModel),
+        // );
+
+        conversationWithTwoResponses =
+          conversationData.prepareModelConversationBasedOnRequests(
+            defaultModel,
+            ['draw smiling emoticon', 'draw a cloud'],
+          );
+        conversationWithTwoResponses.messages[0].custom_content = {
+          attachments: [conversationData.getAttachmentData(imageUrl)],
+        };
+        conversationWithTwoResponses.messages[2].custom_content = {
+          attachments: [conversationData.getAttachmentData(imageUrl2)],
+        };
+
+        conversationData.resetData();
+
+        conversationInFolder =
+          conversationData.prepareConversationWithAttachmentInResponse(
+            imageInConversationInFolderUrl,
+            defaultModel,
+            folderName,
+          );
+
+        conversationData.resetData();
+        conversationWithSpecialChars =
+          conversationData.prepareDefaultConversation(defaultModel);
```

Comment:
why don't you use `prepareHistoryConversationWithAttachmentsInRequest()` or `prepareConversationWithAttachmentsInRequest` ?

Reply:
done